aggregations/db_tables/near_ecosystem_entities.py

```diff
@@ -14,7 +16,12 @@ def sql_create_table(self):
                 website     TEXT, 
                 category    TEXT,    
                 status      TEXT, 
-                contract    TEXT
+                contract    TEXT,
+                logo        TEXT,
+                app         BOOLEAN,
+                nft         BOOLEAN,
+                guild       BOOLEAN 
```

Comment:
This naming looks confusing, I tried to read https://github.com/near/near-analytics/pull/29 and it's so hard to understand what's going on

Let's at least add `is_` prefix to boolean fields.

I personally insist on adding `NOT NULL` in all fields where possible. This is not a blocker for now, but you can't mark this solution as production-ready while we have missing data and no constraints.

Reply:
added is as prefix to booleans.

All this text is coming from the ecosystem repository, some of those is null.  There is nothing wrong with having null values.  Eric needs this data, let him decide course of action